AsyncAwait.Task1.CancellationTokens/Calculator.cs

```diff
@@ -1,19 +1,19 @@
 ﻿using System.Threading;
+using System.Threading.Tasks;
 
 namespace AsyncAwait.Task1.CancellationTokens;
 
 internal static class Calculator
 {
-    // todo: change this method to support cancellation token
-    public static long Calculate(int n /*, CancellationToken token*/)
+    public static async Task<long> CalculateAsync(int n, CancellationToken cancellationToken)
     {
         long sum = 0;
 
         for (var i = 0; i < n; i++)
         {
             // i + 1 is to allow 2147483647 (Max(Int32)) 
             sum = sum + (i + 1);
-            Thread.Sleep(10);
+            await Task.Delay(10);
```

Comment:
You don't need to use Task.Delay() here...
Instead you need to wrap all calde with 
await new Task(() => { your calculation}, cancellationToken).Run();